lib/manager/python/index.js

```diff
@@ -0,0 +1,16 @@
+const { extractDependencies } = require('./extract');
+const { getPackageUpdates } = require('./package');
+const { updateDependency } = require('./update');
+
+const filePattern = new RegExp('^requirements.txt$');
```

Comment:
Is there a rule that `requirements.txt` can be found only in the top level of a repository? i.e. not allowed to nest a python project inside a repo or have multiple `requirements.txt`?